packages/default/scss/popup/_theme.scss

```diff
@@ -20,7 +21,7 @@
         background: $grouping-header-bg;
     }
     .k-popup .k-list .k-item > .k-group::before {
-        border-color: $grouping-header-bg $grouping-header-bg transparent transparent;
+        border-color: $grouping-border;
```

Comment:
This doesn't look right. Previous styles are applied that the element will have border 1px everywhere, but with different color (`border-top/right/bottom/left`). Please, check it.